docs/examples/mcp-reverse-proxy/docker-compose.sse.yaml

```
@@ -0,0 +1,254 @@
+#
+# 4. Development mode (no reverse proxy):
+#    docker compose -f docker-compose.sse.yaml up mcp-server postgres -d
+#
+# 5. View logs:
+#    docker compose -f docker-compose.sse.yaml logs -f mcp-server
+#
+# 6. Stop services:
+#    docker compose -f docker-compose.sse.yaml down
+#
+# 7. Full cleanup:
+#    docker compose -f docker-compose.sse.yaml down -v --remove-orphans
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)